examples/java-export/src/main/java/com/github/bazelbuild/rulesjvmexternal/example/export/Main.java

```diff
@@ -7,8 +7,8 @@
 public class Main {
 
   public static void main(String[] args) {
-    var fromGuava = ImmutableList.of("Hello", "World!");
-    var fromJre = List.of("Hello", "World!");
+    List<String> fromGuava = ImmutableList.of("Hello", "World!");
```

Comment:
The lists returned by `List.of` are immutable. We can replace the guava usage with standard JDK here

Reply:
I thought the point of the example was to use both Guava and JRE Lists and compare them?